lambda_function.py

```diff
@@ -44,27 +44,29 @@ def lambda_handler(event, lambda_context):
                 and (k in message["user"] for k in ("values"))
                 and (k in message["purpose"]["params"] for k in ("platform", "id"))
             ):
+                # sets is_multiple_entry
+                # if the length of user input list has more than 1 userID
+                if len(message["user"]["values"]) > 1:
+                    row["is_multiple_entry"] = True
```

Comment:
Also, as pritam said, instead of this flag, we should have the number of entries

Reply:
Yeah, I'm not actually going to make this part of this PR now. Because I still need to test the whole staging flow. Will make a new PR later.

Reply:
Cool
But you'll make that change before deploying the flow on prod right?
Because it shouldn't be such that some entries in the table are in the column `is_multiple_entry` and some are in `number_of_entries`

Reply:
https://github.com/avantifellows/auth-to-bigquery/pull/4